app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -0,0 +1,33 @@
+<div id="account-recovery-key">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Recovery key{{/t}}</h2>
+
+        <p class="description">
+            <span class="note">{{#t}}NOTE:{{/t}}</span>
+            {{#t}}Keep your recovery key in a safe place where you can access it later. You will not be able to access this page again. You can download or print your key or email it to someone you trust.{{/t}}
```

Comment:
For the last sentence, I'd ditch the `You can` to make it more imperative that the user do something with the key.